CompleteSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java

```diff
@@ -43,6 +43,7 @@
   String PCODE = null;
   String DOMAIN = null;
   private final String ACCOUNT_ID = "accountID";
+  Boolean AUTOPLAY = false;
```

Comment:
it can be a primitive + private modifier

Reply:
@isheftel-ooyala Done